<ui version="4.0" >
 <class>AtomColors</class>
 <widget class="QDialog" name="AtomColors" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Atom Coloring Scheme</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QListWidget" name="listWidget" />
     </item>
     <item>
      <widget class="QToolButton" name="colorToolButton" >
       <property name="text" >
        <string>Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="1" column="1" >
      <widget class="QPushButton" name="deleteTypePushButton" >
       <property name="text" >
        <string>Delete Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QPushButton" name="resetPushButton" >
       <property name="text" >
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2" >
      <widget class="QCheckBox" name="saveCheckBox" >
       <property name="text" >
        <string>Save to preferences</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QPushButton" name="editNamePushButton" >
       <property name="text" >
        <string>Edit Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="addTypePushButton" >
       <property name="text" >
        <string>Add Type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AtomColors</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AtomColors</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>317</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
